### PR TITLE
Bug 1443588 Added info on image tag policy

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1443588

cc: @tdawson 

I'll ask for verification in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
I think this is ready for peer review.

@adellape @ahardin-rh @bmcelvee @gaurav-nelson @mburke5678 I'm very much open to suggestions about this. Let me know if there's ideas for better placement or wording. Thanks!

File: architecture/core_concepts/containers_and_images.adoc [hunk 1]
```diff
@@ -79,11 +79,37 @@ image is referred to uniquely by its hash (a long hexadecimal number
...
 `centos` (implying the `latest` tag), `centos:centos7`, or `fd44297e2ddb`.
 
+[[WARNING]]
+====
+Do not use the `latest` tag for any official {product-title} images. These are
+images that start with `openshift3/`. `latest` can refer to a number of
+versions, such as `3.4`, or `3.5`.
+====
+
+How you tag the images dictates the updating policy. The more specific you are, the less frequently the image will be updated. Use these following to determine the your chosen {product-title} images policy:
```
Comment (ahardin-rh) [inline architecture/core_concepts/containers_and_images.adoc]: s/the your chosen/your chosen

File: architecture/core_concepts/containers_and_images.adoc [hunk 2]
```diff
@@ -79,11 +79,37 @@ image is referred to uniquely by its hash (a long hexadecimal number
...
+The vX.Y tag points to X.Y.Z-<number>. For example, if the `registry-console`
+image is updated to v3.4, it points to the newest 3.4.Z-<number> tag, such
+as 3.4.1-8.
+
+X.Y.Z::
+Similar to the vX.Y example above, the X.Y.Z tag points to the latest
+X.Y.Z-<number>. For example, 3.4.1 would point to 3.4.1-8
+
+X.Y.Z-<number>::
+The tag is unique and does not change. When using this tag, the image does not update if any latest is found. For example, the 3.4.1-8 will always point to 3.4.1-8, even if an image is updated.
```
Comment (ahardin-rh) [inline architecture/core_concepts/containers_and_images.adoc]: if any latest image?

File: architecture/core_concepts/containers_and_images.adoc (hunk 1 above)
Comment (mburke5678) [inline architecture/core_concepts/containers_and_images.adoc]: to choose your {product-title} images policy:  (??)

File: install_config/install/advanced_install.adoc [hunk 3]
```diff
@@ -92,6 +92,15 @@ choose an example that matches your requirements, modify it to match your own
...
 xref:running-the-advanced-installation[running the advanced installation].
 
+[[discrete]]
+[[advanced-install-image-version-policy]]
+=== Image Version Policy
+
+Images require a version number policy in order to maintain updates. See
+the
+xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy][Image
+Version Tag Policy] section in the Architecture Guide for more information.
```
Comment (mburke5678) [inline install_config/install/advanced_install.adoc]: This came up in the AsciiDoc guidelines group. Do we format book titles when they are in a sentence: "in the Architecture Guide"?

File: architecture/core_concepts/containers_and_images.adoc [hunk 4]
```diff
@@ -79,11 +79,37 @@ image is referred to uniquely by its hash (a long hexadecimal number
 e.g. `fd44297e2ddb050ec4f...`) which is usually shortened to 12
 characters (e.g. `fd44297e2ddb`).
 
+[[discrete]]
```
Comment (adellape) [inline architecture/core_concepts/containers_and_images.adoc]: Single square brackets for `[discrete]` here (and 1 more below).

File: install_config/install/advanced_install.adoc (hunk 3 above)
Comment (bfallonf) [inline install_config/install/advanced_install.adoc]: @mburke5678 As in, putting the "Architecture Guide" in italics or something?

"see the _Architecture Guide_ for more information"

**Comment by bfallonf** [comment]
As discussed over IRC, this is fine. I'll merge this. Thanks, all.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../architecture/core_concepts/containers_and_images.adoc#architecture-core-concepts-containers-and-images[Core Concepts -> Containers and Images]
|Added the xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy[Image Version Tag Policy] section.
%

**Pull request merged**